### ISSUE TITLE
Crosshair overlay crashes

### ISSUE THREAD

**Issue by ZombieToof** [Nathan-Schwartz/OW-crosshair#1]
Since a few days the overlay crashes. It seems the simple io plugin does not work anymore.

The current AppData\Local\Overwolf\Log\Apps\X-hair\Crosshair_2017-11-24_17-17.log:

```
28/11/2017 11:17:43.678 <> (/src/crosshair.js:162) Uncaught TypeError: plugin(...).isDirectory is not a function
```

If I add `console.error(plugin())` I get `[object HTMLEmbedElement]`

I can replace the isDirectory() callback with a plain call of `callback(mypath, status)`, but then it fails cause `plugin().LOCALAPPDATA` is `undefined` and mypath iends up as `undefined\Overwolf\Extensions\hcoldgdmlmlajgpfcfipamlekcddjljibbbcodjb\0.0.1`

Even hardcoding the path doesn't help. I found no logs or tracebacks.

If I install the sample app from https://github.com/overwolf/simple-io-plugin it crashes too.
```
28/11/2017 12:36:00.581 <> (/index.html:62) Uncaught TypeError: plugin(...).fileExists is not a function
```

Overwolf is version 0.108.34.0.

**Comment by ZombieToof** [comment]
The plugin seems to have been renamed moved to https://github.com/overwolf/overwolf-plugins and the old repo is deprecated.

**Comment by Nathan-Schwartz** [comment]
Thanks for the detailed issue writeup 👍 

It looks like this is related to the deprecation of the old plugins late last year.
> all apps on the Overwolf Appstore that use NPAPI plugins should be updated to use our new C# plugin technology.

Sorry for letting this repo slide, I haven't kept up with my Overwolf apps as much as I intended to.
I am not sure when I'll get a chance to work on a fix, but I would be happy to accept a PR if you are up for it! 🙂

**Comment by Apx99** [comment]
I know its only been a month or so, but is this issue being looked into?

**Comment by Nathan-Schwartz** [comment]
@Re-Venge I no longer have a windows computer I can develop Overwolf apps on, so unfortunately I won't be able to resolve this in a timely manner. I commented above saying what I think the issue is, but I haven't been able to test my theory.

If someone decides to get it working, I'd love a PR 🙂

**Comment by Nathan-Schwartz** [comment]
@Re-Venge @ZombieToof this should be resolved by #2 thanks to @Mennims 


If the issue persists, feel free to reopen
